features/no-disk-space.feature

```diff
@@ -0,0 +1,16 @@
+Feature: No Disk Space Overlay
+
+  Background:
+    Given I have completed the basic setup
+
+  Scenario: I set the required space to 1 KB
+    Given I set the required space to 1 KB
+    And I check the disk space
+    Then The No Disk Space overlay should be hidden
+
+  Scenario: I set the required space to 100 TB
+    Given I set the required space to 100 TB
+    And I check the disk space
+    Then The No Disk Space overlay should be visible
+    And After 10 seconds
```

Comment:
@daniloprates this test step should be replaced with a more descriptive step such as "After I free up disk space".

Reply:
What actually happens after the 10 seconds is that it will check using the default `DISK_SPACE_REQUIRED` param, therefore I replaced this step with `I set the required space to 1 KB`.